worlds/ff6wc/__init__.py

```diff
@@ -454,7 +457,9 @@ def set_rules(self):
 
         for check in Locations.major_checks:
             add_item_rule(self.get_location(check),
-                          lambda item: item.name not in Items.okay_items)
+                          # add things that are NOT in the non-rewards list
+                          # or something for another player (based on their settings)
+                          lambda item: item.name not in self.item_nonrewards or item.player != self.player)
```

Comment:
Do you know what the purpose of this item rule is?
I think maybe we talked about it before (why you made this change to ` or item.player != self.player`) but I don't remember.

I'm not seeing any good reason for the rule to exist at all, and it's causing generation errors.
It seems like it defeats the purpose of Treasuresanity (and doesn't do anything when not in Treasuresanity).

Reply:
I was getting generation errors without the change that I made to the item.player != self.player, but if you're not seeing it, then I guess it was something wrong I was doing on my end.

Reply:
I think the change you made just reduced the probability of the errors, so you were less likely to see it.
I decided to just remove the rule, since I don't think it's needed.